pipeline/beam_tables.py

```diff
@@ -609,24 +534,24 @@ def run_beam_pipeline(self, scan_type: str, incremental_load: bool,
       lines = _read_scan_text(p, new_filenames)
 
       if scan_type == satellite.SCAN_TYPE_SATELLITE:
-        # PCollection[Row], PCollection[Row]
+        # PCollection[SatelliteRow], PCollection[BlockpageRow]
         satellite_rows, blockpage_rows = satellite.process_satellite_lines(
             lines)
 
-        # PCollection[Row]
+        # PCollection[SatelliteRow]
         rows_with_metadata = self._add_metadata(satellite_rows)
 
         self._write_to_bigquery(
             satellite.SCAN_TYPE_BLOCKPAGE, blockpage_rows,
             satellite.get_blockpage_table_name(table_name, scan_type),
             incremental_load)
       else:  # Hyperquack scans
-        # PCollection[Row]
+        # PCollection[HyperquackRow]
         rows = (
             lines | 'flatten json' >> beam.ParDo(
                 flatten.FlattenMeasurement()).with_output_types(Row))
```

Comment:
Do you need this?
```suggestion
                flatten.FlattenMeasurement()).with_output_types(HyperquackRow))
```

Reply:
Yes, the hyperquack and satellite flows are different. I'm not a big fan of how they're intertwined here, but that's not that I'm trying to fix in this change.